README.md

```diff
@@ -35,64 +35,14 @@
 
 ## Deployment
 
-The `staging` branch is used to deploy this website to
-https://staging.gazebosim.org. The `production` branch is used to deploy
-this website to `https://gazebosim.org`.
+The `staging` branch in this repository is used to deploy this website to
```

Comment:
Is pushing directly to `staging` the recommended workflow or should we create PRs? How about for `production`?

Reply:
I added a note in [1528ff1](https://github.com/gazebo-web/gazebosim-web-frontend/pull/14/commits/1528ff1505c5ab1dbeb108b202624940ec00257a)